src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -48,6 +48,7 @@ private class InvocationState
         public UInt160 CallingScriptHash => CurrentContext?.GetState<ExecutionContextState>().CallingScriptHash;
         public UInt160 EntryScriptHash => EntryContext?.GetState<ExecutionContextState>().ScriptHash;
         public IReadOnlyList<NotifyEventArgs> Notifications => notifications ?? (IReadOnlyList<NotifyEventArgs>)Array.Empty<NotifyEventArgs>();
+        internal HashSet<OracleResponse> OracleResponses { get; set; }
```

Comment:
HashSet ？ The order will be changed.

Reply:
Do you think that the order it's important? it will be used for remove data and mint